feedback-module/public/locales/en/translation.json

```diff
@@ -20,58 +20,49 @@
     ]
   },
 
-  "missingBtn": "I couldn't find what I was looking for",
-  "brokenBtn": "Something appears to be broken or inaccurate",
-  "otherBtn": "I havce other feedback on this page or website",
-
-  "missingTitle": "Sorry you couldn't find what you were looking for! Can you tell us a little more?",
-  "missingCheckboxes": [
-    "I wasn't sure where to look",
-    "I was confused by the information I found",
-    "I don't understand how to use this website",
-    "I hit a dead end",
-    "The content was not available in my preferred language",
-    "There were errors with the translation available",
-    "Other"
-  ],
-  "missingTextInputs": [
-    { "type": "text", "text": "What were you looking for or trying to do?" },
-    { "type": "text", "text": "Would you like to add anything else?" }
-  ],
-
-  "brokenTitle": "Sorry to hear that something is broken or inaccurate! Can you tell us a little more?",
-  "brokenCheckboxes": [
-    "There is a spelling mistake or typo",
-    "A link or button does not work",
-    "The information is out of date",
-    "I got different informatino from another source (311, printed mailer, elected official, etc.)",
-    "The layout looks broken",
-    "I got an error",
-    "Other"
-  ],
-  "brokenTextInputs": [
-    {
-      "type": "text",
-      "text": "Can you give us more details on what you were trying to do, or the exact problem you encountered? (optional)"
-    }
-  ],
+  "broken": {
+    "initialBtn": "Something appears to be broken or inaccurate",
```

Comment:
Could we just call this button instead of initialBtn?

Reply:
I thought it might be confusing as there is also a button on the actual broken page (the submit button)

Reply:
Hmmm that makes sense!